docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,243 @@
+(including the master builds) produces an ```ipxe.cfg``` file and publishes all the required artifacts on GitHub
+in the tagged release assets area. The same set of artifacts can be obtained locally by running
+```docker run lfedge/eve installer_net | tar xf -```. Regardless of whether you're using
+artifacts published on GitHub or on your local http server, as long as iPXE can successfully
+resolve and access their URLs, the process will work. The default ```ipxe.cfg```
+file published by EVE-OS releases assumes GitHub URLs. Therefore it needs to be edited 
+if you are publishing files and deploying in a local environment. 
+Here's an example of a customized iPXE configuration file used to run an EVE-OS
+installer image locally in a hardware lab. Note the changes made to both ```url``` and ```eve_args``` variables.
+
+```console
+#!ipxe
+# set url https://github.com/lf-edge/eve/releases/download/snapshot/amd64.
+set url https://10.0.0.2/eve/releases/download/snapshot/amd64.
+# set eve_args eve_soft_serial=${ip} eve_reboot_after_install
+set eve_args eve_soft_serial=${ip} eve_install_server=zedcontrol.hummingbird.zededa.net eve_reboot_after_install
+
+# you are not expected to go below this line
+set console console=ttyS0 console=ttyS1 console=ttyS2 console=ttyAMA0 console=ttyAMA1 console=tty0
+set installer_args root=/initrd.image find_boot=netboot overlaytmpfs fastboot
+
+# you need to be this ^ tall to go beyound this point
+kernel ${url}kernel ${eve_args} ${installer_args} ${console} ${platform_tweaks} initrd=amd64.initrd.img initrd=amd64.installer.img initrd=amd64.initrd.bits initrd=amd64.rootfs.img
+initrd ${url}initrd.img
+initrd ${url}installer.img
+initrd ${url}initrd.bits
+initrd ${url}rootfs.img
+boot
+```
+
+Most of the time you wouldn't need to edit ```ipxe.cfg``` since the default values provide
+adequate out-of-the-box behavior of the installer. In fact, you can simply supply the URL
+of the ```ipxe.cfg``` file published on GitHub as the only input to the iPXE executable. 
+This method essentially lets you boot a server you don't even have access to off of EVE-OS releases on GitHub!
+
+If you don't have a physical server to test EVE-OS installation you can try installing it on an "iPXE custom server" from 
+Equinix Metal. Equinix Metal offers both a web UI and a command-line interface approach to 
+defining the software that you would like to run on one of their servers. (As a prerequisite you must 
+have an Equinix Metal account, access to a project, and have uploaded your personal ssh key.)
+
+Here's an example of the steps involved to install EVE-OS using the Equinix Metal web UI:
+* Servers => On Demand
+* Select Location => e.g., Silicon Valley = sjc1
+* Select Your Server => e.g., the cheapest one to rent is t1.small.x86
+* Select an Operating System => Custom iPXE
+* IPXE Script URL => https://github.com/lf-edge/eve/releases/download/snapshot/amd64.ipxe.efi.ip.cfg
```

Comment:
It is really needed to be mentioned here that the "snapshot" image will try to onboard to `alpha` controller

Reply:
yup -- how would you know otherwise?